src/components/database/settings/driver-config/EditDriverConfigForm.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Modal, Form, Input } from 'antd';
+import FormItemLabel from '../../../form-item-label/FormItemLabel';
+
+const EditDriverConfigForm = ({ handleSubmit, handleCancel, dbType, initialValue }) => {
+
+  const [form] = Form.useForm()
+
+  const handleSubmitClick = () => {
+    form.validateFields().then(values => {
+      handleSubmit(values)
```

Comment:
All the values in the `driverConf` are supposed to be numbers. Make sure they are sent as numbers rather than strings. This is the right place to do that conversion.